target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -11,6 +11,13 @@ board_config_update
 board=$(ipq806x_board_name)
 
 case "$board" in
+ad7200)
+	ucidef_set_led_usbdev "usb1" "USB 1" "${board}:blue:usb_1" "2-1"
+	ucidef_set_led_usbdev "usb2" "USB 2" "${board}:blue:usb_3" "4-1"
+	ucidef_set_led_netdev "wan" "WAN" "${board}:blue:wan" "eth0"
+	ucidef_set_led_netdev "lan" "LAN" "${board}:blue:lan" "br-lan"
+	ucidef_set_led_default "general" "general" "${board}:blue:status" "1"
```

Comment:
Drop this. This LED is already handled by diag.sh due to the led-running = &power; in your dts.